client/messaging/message-input.tsx

```diff
@@ -69,6 +91,17 @@ export interface MessageInputProps {
    * the user's ID to handle user changing their account.
    */
   storageKey?: string
+  /**
+   * An optional list of users that can be mentioned in the message input. If provided, the message
+   * input will display a popover with all matching users when the user starts typing something
+   * *after* the @ character and there's a match.
+   */
+  mentionableUsers?: SbUser[]
+  /**
+   * Similar to the `mentionableUsers` property above, except this list will be used when the user
+   * has only typed the @ character and nothing else after it.
+   */
+  defaultMentionableUsers?: SbUser[]
```

Comment:
I think this naming is somewhat weird since `defaultFoo` is usually for uncontrolled values. Come up with something else